frontend/packages/cli/index.html

```diff
@@ -5,6 +5,12 @@
     <link rel="icon" href="/public/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Liam ERD</title>
+    <link
+      rel="preload"
+      href="/loading.riv"
+      as="fetch"
+      crossorigin="anonymous"
+    />
```

Comment:
📝 Essentially unnecessary, but preloaded to speed up loading.
ref: [Optimization techniques for Rive animations in React apps - DEV Community](https://dev.to/alex_barashkov/optimization-techniques-for-rive-animations-in-react-apps-1a8p)